packages/calling/src/CallingClient/registration/types.ts

```diff
@@ -9,18 +9,98 @@ export type restoreRegistrationCallBack = (
   caller: string
 ) => Promise<boolean>;
 
+/**
+ * Represents an interface for managing registration-related operations.
+ */
 export interface IRegistration {
-  setMobiusServers: (primaryMobiusUris: string[], backupMobiusUris: string[]) => void;
-  triggerRegistration: () => Promise<void>;
-  isDeviceRegistered: () => boolean;
-  setStatus: (value: MobiusStatus) => void;
-  getStatus: () => MobiusStatus;
-  getDeviceInfo: () => IDeviceInfo;
-  clearKeepaliveTimer: () => void;
-  deregister: () => void;
-  setActiveMobiusUrl: (url: string) => void;
-  getActiveMobiusUrl: () => string;
-  reconnectOnFailure: (caller: string) => Promise<void>;
-  isReconnectPending: () => boolean;
-  handleConnectionRestoration: (retry: boolean) => Promise<boolean>;
+  /**
+   * Sets the primary and backup Mobius server URLs.
+   *
+   * @param primaryMobiusUris - An array of primary Mobius server URLs.
```

Comment:
In this file as well, we can just keep the descriptions. @ignore and @param might not be required

Reply:
Will retain @param in this file for internal reference.